test-cljs/com/climate/geojson_schema/test/resources.clj

```diff
@@ -0,0 +1,16 @@
+(ns com.climate.geojson-schema.test.resources
+  (:require [clojure.java.io :refer [resource file]]
+            [cheshire.core :as json]))
+
+(defmacro all-resources
+  "Eagerly pulls in all of the geojson_examples.
+  Pass in the symbol to def in code.
+  It will contain a map short-file -> file-data"
+  [symbol]
+  (let [data (->> (resource "geojson_examples")
+                  file
+                  file-seq
+                  (remove #(.isDirectory %))
+                  (map (juxt #(.getName %) #(json/parse-string (slurp %) true)))
+                  (into {}))]
+    `(def ~symbol ~data)))
```

Comment:
Seems like the use of a macro here isn't really required. Would be better if ```all-resources``` was just a function without side effects - the called can choose to ```def``` the result in the appropriate namespace.